lib/can_do.rb

```diff
@@ -38,18 +40,24 @@ def self.features
     instance.features
   end
 
+  def self.shared_features(name)
+    CONNECTION_POOL.with do |redis|
+      redis.get(FEATURE_KEY_PREFIX+name.to_s)
```

Comment:
how does this behave if Redis is (for some reason) unresponsive? try switching off your Redis server - and if it crashes, let's catch the error in here, to make this code fault-tolerant (otherwise, it has the potential to crash entire requests).

Reply:
while testing chronos, i realised that I need to add `CANDO_REDIS_URL`. 
I guess we need to update README for that. any services that is using can-do should update travis and ansible script.